torch/csrc/StorageMethods.cpp

```diff
@@ -62,6 +62,13 @@ static PyObject* THPStorage_dataPtr(PyObject* self, PyObject* noargs) {
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* THPStorage_resizable(PyObject* self, PyObject* noargs) {
+  HANDLE_TH_ERRORS
+  THPStorage_assertNotNull(self);
+  return PyBool_FromLong(THPStorage_Unpack(self).resizable());
```

Comment:
Why `PyBool_FromLong`, (might have missed something but doesn't `.resizable()` return `bool`?)

https://github.com/pytorch/pytorch/blob/238d87f74df7cbf38ce55c89abd18c5234ee55c7/c10/core/Storage.h#L101-L103

Reply:
I mean, there's no PyBool_FromBool lol, cuz bool isn't a thing in C haha.

Reply:
oh whoops, got it!